packages/perseus/src/components/sortable.tsx

```diff
@@ -859,22 +846,20 @@ class Sortable extends React.Component<SortableProps, SortableState> {
                         item.endPosition = endPosition;
                     }
                     return item;
-                },
-                this,
-            );
+                })
+                .filter(isDefined);
 
             this.setState({items: items});
             // HACK: We need to know *that* the widget changed, but currently it's
             // not set up in a nice way to tell us *how* it changed, since the
             // permutation of the items is stored in state.
-            this.props.onChange && this.props.onChange({});
+            this.props.onChange({});
```

Comment:
This was another thing that `@typescript-eslint/no-unnecessary-condition` flagged.